.github/workflows/development-environment.yml

```diff
@@ -74,8 +74,25 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}
 
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
       - name: Set up development environment (as per docs)
+        shell: bash
+        # XXX: I don't understand why we need to source again when we already do in a previous step
```

Comment:
State is not maintained between steps, I believe it opens a new shell

Reply:
That's correct.

Reply:
@armenzg Could you remove this comment?